ClemBot.Bot/bot/api/emote_board_route.py

```diff
@@ -0,0 +1,257 @@
+from typing import Any, Union, cast
+
+import discord
+
+from bot.api.api_client import ApiClient
+from bot.api.base_route import BaseRoute
+from bot.models.emote_board_models import (
+    EmoteBoard,
+    EmoteBoardPost,
+    EmoteBoardReaction,
+    PopularLeaderboardSlot,
+    PostLeaderboardSlot,
+    ReactionLeaderboardSlot,
+)
+from bot.utils.logging_utils import get_logger
+
+MIN_LIMIT = 1
+DEF_LIMIT = 5
+MAX_LIMIT = 50
+
+log = get_logger(__name__)
+
+
+class EmoteBoardRoute(BaseRoute):
+    def __init__(self, client: ApiClient):
+        super().__init__(client)
```

Comment:
Nothing extra is happening in this `__init__`, we're overriding it for seemingly no reason

Reply:
BaseRoute requires it. We reflect for baseroute instances to load routes dynamically